### PR TITLE
Updating "Default.sublime-commands" to be up-to-date with the Build 4152

### PR DESCRIPTION
**Pull request by d0vgan**
Updating "Default.sublime-commands" to be up-to-date with the Build 4152.
Adding a simple script "subl_def_commands.py" for "Default.sublime-commands" files comparison.

### CODE REVIEW

**Comment by michaelblyons** [comment]
Can you skip the ones that are just white space?

File: Default.sublime-commands [hunk 1]
```diff
@@ -67,6 +67,8 @@
     { "caption": "Toggle Comment", "command": "toggle_comment", "args": {"block": false} },
     { "caption": "Toggle Block Comment", "command": "toggle_comment", "args": {"block": true} },
 
+    { "caption": "Wrap at Ruler", "command": "wrap_lines" },
+
```
Comment (deathaxe) [inline Default.sublime-commands]: Already added it earlier, but forgot to push. It's now in master.

File: Default.sublime-commands [hunk 2]
```diff
@@ -126,7 +129,7 @@
 
     // Code Folding
     { "caption": "Code Folding: Fold", "command": "fold" },
-    { "caption": "Code Folding: Fold All", "command": "fold_all" },
+    { "caption": "Code Folding: Fold All", "command": "fold_by_level", "args": {"level": 1} },
```
Comment (deathaxe) [inline Default.sublime-commands]: I don't think I want to make this change.

Instead adding an explicit "Fold by Level 1" is preferable.

```
{ "caption": "Code Folding: by Level 1", "command": "fold_by_level", "args": {"level": 1} },
```

File: subl_def_commands.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+def run():
+    cmds1 = read_commands_from_file("Default.sublime-commands")
+    cmds2 = read_commands_from_file("SublimeText/Data/Packages/Default/Default.sublime-commands")
+    for cmd in cmds1:
+        if cmd not in cmds2:
+            print(cmd)
+
+
+run()
```
Comment (deathaxe) [inline subl_def_commands.py]: I prefer tracking changes of Default package in a local repo. This file is not needed upstream.

**Review by deathaxe — CHANGES_REQUESTED** [review]
Whitespace changes make sense to me as they improve code quality.

Would however suggest some other changes.

File: Default.sublime-commands (hunk 2 above)
Comment (d0vgan) [reply Default.sublime-commands]: This corresponds to the "official" content of "Default.sublime-commands", lines 136-138:
```
	{ "caption": "Code Folding: Fold", "command": "fold" },
	{ "caption": "Code Folding: Fold All", "command": "fold_by_level", "args": {"level": 1} },
	{ "caption": "Code Folding: Unfold", "command": "unfold" },
```

File: Default.sublime-commands (hunk 1 above)
Comment (d0vgan) [reply Default.sublime-commands]: Do you want me to remove this line from the pull request?

File: Default.sublime-commands (hunk 2 above)
Comment (deathaxe) [reply Default.sublime-commands]: I am aware of that, but it doesn't mean to apply it under all circumstances in this repo.

Some users may prefer the old behavior. See also: https://forum.sublimetext.com/t/st4-code-folding-level-1-issues/69472/2

File: Default.sublime-commands (hunk 1 above)
Comment (deathaxe) [reply Default.sublime-commands]: It would add a duplicated item when merged, so yes.

Comment (d0vgan) [reply Default.sublime-commands]: done!

File: Default.sublime-commands (hunk 2 above)
Comment (d0vgan) [reply Default.sublime-commands]: done!

File: subl_def_commands.py (hunk 3 above)
Comment (d0vgan) [reply subl_def_commands.py]: done!

**Review by deathaxe — APPROVED** [review]

**Comment by deathaxe** [comment]
Thanks!

**Pull request merged**